server/routes/index.js

```diff
@@ -582,6 +583,8 @@ module.exports = function Index ({ authenticationMiddleware }) {
 
   router.use('/:courtCode/outcomes', outcomesRouter)
 
+  router.post('/hearing/:hearingId/outcome/assign', defaults, catchErrors(assignHearingOutcomeRouteHandler))
```

Comment:
This can go to outcomes router here https://github.com/ministryofjustice/prepare-a-case/blob/e98ff8ed17b8e5089a67874e192ad2a13300acff/server/routes/handlers/outcomes/outcomesRouter.js#L16